src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -100,7 +101,8 @@ public function reveal(string $name): ?string
         $this->lastMessage = null;
         $this->validateName($name);
 
-        if (!is_file($file = $this->pathPrefix.$name.'.'.substr_replace(md5($name), '.php', -26))) {
+        $filename = $this->getFilename($name);
```

Comment:
let's use `$file` everywhere as it was the case, this will prevent needles cs diff and fix L123/125 below

Reply:
@nicolas-grekas I haven't changed `$file`:
* `$file` is still the full path to the file
* `$filename` is the name of the file without the extension (as per https://www.php.net/manual/en/function.pathinfo.php)

The new method `getFilename()` defines in a single spot this filename to prevent duplication. Before, there were 2 different definitions:
- 2x `$name.'.'.substr_replace(md5($name), '.php', -26)`
- 1x `$name.'.'.substr(md5($name), 0, 6)`

=> Now it's `$this->pathPrefix.$filename.'.php'` everywhere and there is a single definition of `$filename`.

I don't understand your comment about L123/125 🤔

Reply:
I missread the code, LGTM.